### PR TITLE
Include the examples in the built sources

### PR DESCRIPTION
**Pull request by mapreri**
Thank you ♥

### CODE REVIEW

**Comment by remi-stripe** [comment]
@mapreri Thanks for the PR! We'll have a look and let you know if we need anything else!

**Comment by richardm-stripe** [comment]
Hi @mapreri, this looks good to me. Out of curiosity, what rationale/use case do you have for this, or what led you to open this PR?

**Comment by mapreri** [comment]
> Hi @mapreri, this looks good to me. Out of curiosity, what rationale/use case do you have for this, or what led you to open this PR?

I was building a Debian package for internal use at a customer and being the Debian Developer I am I try hard to follow our customs, which includes keeping as much documentation as offline as possible, including the examples.  Indeed I tend to look up code locally than go and google about whatever I need.  Since I found the examples small and useful I wished to have them near me.

**Comment by richardm-stripe** [comment]
That makes sense. Thanks for the contribution! It'll go out in the next release.
